<launch>
  <arg name="input_image" default="/kinect2_head/rgb/image" />
  <arg name="input_info" default="/kinect2_head/rgb/camera_info" />
  <arg name="respawn" default="false" />

  <node pkg="imagesift" type="imagesift" name="imagesift">
    <remap from="image" to="$(arg input_image)" />
    <remap from="camera_info" to="$(arg input_info)" />
  </node>
  <node pkg="posedetection_msgs" type="feature0d_to_image" name="feature0d_to_image" output="log" />

  <node pkg="jsk_2015_05_baxter_apc" type="bof_object_matcher.py" name="bof_object_matcher" output="screen" respawn="$(arg respawn)">
    <remap from="~input" to="ImageFeature0D" />
  </node>
</launch>